aws/aws_gov.go

```diff
@@ -0,0 +1,42 @@
+package aws
+
+import (
+	"os"
+
+	ctx "github.com/hortonworks/cloud-haunter/context"
+	"github.com/hortonworks/cloud-haunter/types"
+	log "github.com/sirupsen/logrus"
+)
+
+var awsGovProvider = awsProvider{}
+
+func init() {
```

Comment:
This almost the same as aws init. Does it make sense to make a common function for them?